lib/iris/tests/unit/fileformats/grib/load_convert/test_product_definition_template_11.py

```diff
@@ -0,0 +1,112 @@
+# (C) British Crown Copyright 2016, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Test function
+:func:`iris.fileformats.grib._load_convert.product_definition_template_11`.
+
+"""
+
+from __future__ import (absolute_import, division, print_function)
+from six.moves import (filter, input, map, range, zip)  # noqa
+
+# import iris tests first so that some things can be initialised
+# before importing anything else.
+import iris.tests as tests
+
+from copy import deepcopy
+import warnings
+
+from iris.coords import DimCoord, CellMethod
+from iris.fileformats.grib._load_convert import product_definition_template_11
+from iris.tests import mock
+
+
+class Test(tests.IrisTest):
+    def setUp(self):
+        module = 'iris.fileformats.grib._load_convert'
+        self.patch('warnings.warn')
+        this = '{}.product_definition_template_11'.format(module)
```

Comment:
This could probably do with giving it a helpful name i.e. 'target' or 'target_pdt'